tests/test_integration_http.py

```diff
@@ -413,3 +415,60 @@ def delayed(req):
     run(tmp_path, "sentry_example", ["log", "no-setup"], check=True, env=env)
 
     assert len(httpserver.log) == 10
+
+
+def test_transaction_only(cmake, httpserver):
+    tmp_path = cmake(["sentry_example"], {"SENTRY_BACKEND": "none"})
+
+    httpserver.expect_oneshot_request(
+        "/api/123456/envelope/",
+        headers={"x-sentry-auth": auth_header},
+    ).respond_with_data("OK")
+    env = dict(os.environ, SENTRY_DSN=make_dsn(httpserver), SENTRY_RELEASE="🤮🚀")
+
+    run(
+        tmp_path,
+        "sentry_example",
+        ["log", "capture-transaction"],
+        check=True,
+        env=env,
+    )
+
+    assert len(httpserver.log) == 1
+    output = httpserver.log[0][0].get_data()
+    envelope = Envelope.deserialize(output)
+
+    # Show what the envelope looks like if the test fails.
+    envelope.print_verbose()
+
+    # The transaction is overwritten.
+    assert_meta(envelope, transaction="little.teapot")
+
+    # Extract the one-and-only-item
+    (event,) = envelope.items
+
+    assert event.headers["type"] == "transaction"
+    json = event.payload.json
+
+    # See https://develop.sentry.dev/sdk/performance/trace-context/#trace-context
+    trace_context = json["contexts"]["trace"]
+
+    assert (
+        trace_context["op"] == "Short and stout here is my handle and here is my spout"
+    )
+
+    assert trace_context["trace_id"]
+    trace_id = uuid.UUID(hex=trace_context["trace_id"])
+    assert trace_id
+
+    # TODO: currently missing
+    # assert trace_context['public_key']
+
+    assert trace_context["span_id"]
+    assert trace_context["status"] == "ok"
+
+    RFC3339_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+    start_timestamp = time.strptime(json["start_timestamp"], RFC3339_FORMAT)
+    assert start_timestamp
+    timestamp = time.strptime(json["timestamp"], RFC3339_FORMAT)
+    assert timestamp >= start_timestamp
```

Comment:
I think this should be fine, as the example runs all these on the main thread, however we did run into the issue previously that our time source is not guaranteed to be fully monotonic in multithreaded scenarios :-(

Reply:
hum, yes i did worry about this.  if it turns out to be flaky i'll not hesitate the assert though

Reply:
huh "to remove the assert"